test/fixtures/scale.radialLinear/gridlines-scriptable.js

```diff
@@ -11,10 +11,10 @@ module.exports = {
 					gridLines: {
 						display: true,
 						color: function(context) {
-							return context.index % 2 === 0 ? 'red' : 'green';
+							return context.index % 2 === 0 ? 'green' : 'red';
```

Comment:
Why was this change needed?

Reply:
scriptable and indexable had different `index` base, which is not supported anymore